README.md

```diff
@@ -110,6 +110,50 @@ vars:
 
 For more configuration information, see the relevant connectors ([listed above](https://github.com/fivetran/dbt_ad_reporting/edit/master/README.md#adreporting)).
 
+### Changing the Build Schema (RECOMMENDED)
+By default this package will build all models in your <target_schema>.  This behavior can be tailored to your preference by making use of custom schemas. We highly recommend use of custom schemas for this package, as multiple sources are involved.  To do so, add the following configuration to your `dbt_project.yml` file: 
+
+```yml
+# dbt_project.yml
+
+...
+models:  
+  pinterest:
```

Comment:
Should the `enabled: false` still be present for the Pinterest configs here?

Reply:
good call, nope they shouldn't. I've removed.